imports/plugins/core/ui/client/components/badge/__tests__/__snapshots__/badge.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`Badge snapshot test 1`] = `
 <span
-  className="rui badge"
+  className="rui badge Classes to apply"
```

Comment:
I added the `className` prop to the `Badge` component so that I could style one of the badges on the order list a bit differently. Since the `Badge` component has tests written for it, the tests were failing at first because of the prop I had added. On running the tests locally and seeing it fail, there was a suggestion to update the snapshot for the badge test (the file above) by running `npm test -- -u`, which on running it, added the bit of code above, thus fixing the failing tests.